build.gradle

```diff
@@ -12,6 +12,8 @@ import com.github.jk1.license.render.TextReportRenderer
 
 if (project.hasProperty('isTagTriggered')) {
     version = getAbbreviatedGitVersion()
+} else if (project.hasProperty('isEapBuild')){
```

Comment:
Added this property for integration tests to set the version to the `eap` format.